.drone.yml

```diff
@@ -304,6 +304,9 @@ steps:
   - git config --global user.email drone@metrumrg.com
   - git config --global user.name Drony
   - git fetch --tags
+  # clone nmrec from main branch for building package
+  - git clone --depth 1 https://github.com/metrumresearchgroup/nmrec.git /tmp/nmrec
+  - R -s -e 'devtools::install("/tmp/nmrec", upgrade = "never")'
   - R -s -e 'pkgpub::create_tagged_repo(.dir = '"'"'/ephemeral'"'"')'
```

Comment:
As I mentioned out of band last Friday: it'd be better for this to use 

```
devtools::install_dev_deps(upgrade = "never")'
```

because this step should not assume that all of the dependencies are baked into the image.  And, for the purposes of the build, the nmrec version on MPN is good enough, so there's no need for the clone/install from the GitHub repo.

Reply:
Sorry didnt realize you responded offline! Just read through all your comments; updating now

Reply:
No worries.

[ For posterity: the tag pointing to a311e37 was already pushed and triggered the S3 upload, so @barrettk and I restored the tip of this PR to there.  There was a new tag pushed pointing to c65f9ef, but we canceled it before the release step was triggered.  We'll restore that original tag. ]